tests/installation/await_install.pm

```diff
@@ -125,6 +125,13 @@ sub run {
     }
     # on s390 we might need to install additional packages depending on the installation method
     if (check_var('ARCH', 's390x')) {
+      # For poo#93949, we need password possibility
+      if (!is_sle && !is_leap & !is_sle_micro) {
+        select_console 'install-shell';
+        assert_script_run('mountpoint /mnt && mkdir -p /mnt/etc/ssh/sshd_config.d');
+        assert_script_run('echo PermitRootLogin yes > /mnt/etc/ssh/sshd_config.d/allow-root-with-password.conf');
+        select_console 'installation';
+      }
```

Comment:
please extract this into a function. The run method is already too big and complicated

Reply:
Extracted into a function. Test verification is running at the moment...

Reply:
Tests are passed again. :)

Reply:
ok, this part looks good